transport/system.go

```diff
@@ -99,19 +100,25 @@ func (t *System) buildOpenCmd() {
 
 // Open open a standard ssh connection.
```

Comment:
that is a repo-wide thing, but I only mention it here for now. 
typically in docstrings you do "Open open**s**..." like you'd write it

Reply:
resolved..... for this file... all things in time I guess 😁